README.md

```diff
@@ -679,6 +679,17 @@ Offline Installations   | ✓     | x   | x   | x    |
   they could find ways to do an offline install for any package manager, but only `gomod` supports
   this out of the box (i.e. the user does not need to change their workflow).
 
+#### Limitations
+
+Cachito collects all dependencies from repository regardless of whether they used by a certain image
+or not.
+For example, if some repository contains global `go.mod` file, dependencies from the file would be
+used for all images built from the repository (see [gomod](#gomod) for more details).
+
+Meanwhile, Cachito still allows distinguishing between multiple applications from one source using
+**Multiple Paths** feature. So it could be used as a workaround in that case (depends on the
+reporitory structure).
```

Comment:
+1, IMO this is not just a workaround but the proper solution. If you're building 2 applications from your repo, split up your repo into 2 packages.

It's not always in the users' control (e.g. building from an upstream that they do not own), but it's not something Cachito should have to do.

Reply:
Replaced 'workaround' with 'solution'.